service/build.go

```diff
@@ -37,7 +38,7 @@ type Dependencies struct {
 }
 
 // ConstructDependencies builds the blobstore dependencies from a configuration.
-func constructDependencies(cfg *config.Config) (*Dependencies, error) {
+func constructDependencies(cfg *config.Config, HTTPTimeout time.Duration) (*Dependencies, error) {
```

Comment:
I didn't bother documenting the args here because there was just one arg that had its own docs, but we should doc them now since there's two and it's not clear what the timeout applies to